spynnaker/pyNN/models/neural_projections/synapse_information.py

```diff
@@ -13,21 +13,35 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+from spinn_front_end_common.utilities.globals_variables import get_simulator
+
 
 class SynapseInformation(object):
     """ Contains the synapse information including the connector, synapse type\
         and synapse dynamics
     """
     __slots__ = [
         "__connector",
+        "__pre_population",
+        "__post_population",
+        "__prepop_is_view",
+        "__postpop_is_view",
+        "__rng",
         "__synapse_dynamics",
         "__synapse_type",
         "__weight",
         "__delay"]
 
-    def __init__(self, connector, synapse_dynamics, synapse_type,
+    def __init__(self, connector, pre_population, post_population,
+                 prepop_is_view, postpop_is_view, rng,
```

Comment:
Can probably drop the RNG in PyNN8.  Either a connector needs and has an rng parameter or it isn't needed...